packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -7,16 +7,15 @@ Before you can sign a user in, you need to create an Okta app integration that r
 
     <a href="https://developer.okta.com/login" target="_blank" class="Button--blue">Go to Admin Console</a>
 
-1. From the Admin Console side navigation, click **Applications** > **Applications**.
-1. Click **Add Application** and then **Create New App**.
-1. Pick **Single-Page App (SPA)** as the platform and click **Create**.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. Select **OIDC - OpenID Connect** as the **Sign-in method**.
+1. Select **Single-Page Application** as the **Application type** and click **Next**.
     > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

Comment:
```suggestion
    > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, and would break the sign-in or sign-out flow.
```